plugins/mac-capture/mac-screen-capture.m

```diff
@@ -1334,6 +1334,30 @@ - (void)stream:(SCStream *)stream
 	}
 }
 
+- (void)stream:(SCStream *)stream didStopWithError:(NSError *)error
+{
+	NSString *errorMessage;
+	switch (error.code) {
+#if __MAC_OS_X_VERSION_MAX_ALLOWED >= 130000
+	case SCStreamErrorUserStopped:
+		errorMessage = @"User stopped stream.";
+		break;
+#endif
+	case SCStreamErrorNoCaptureSource:
+		errorMessage =
+			@"Stream stopped with no capture source. Did the computer go to sleep?";
```

Comment:
Error messages are not the place to voice hunches or assumption, they should be matter-of-fact only.

If context is required, additional context should be logged instead (i.e. the `NSError` object's error description itself or the stack).

Reply:
I'll just remove it then, there is no more context to add from the NSError side as far as I can tell. The localized description just says "No displays or windows found to record" (original message on my machine: "Keine Displays oder Fenster für die Aufnahme gefunden"), and the stack is not very helpful helpful:
<img width="524" alt="image" src="https://github.com/obsproject/obs-studio/assets/59806498/8f0d2563-0816-4bbb-acd3-f7dbd266d93d">

Reply:
Bummer, but yeah that's kinda what the error code already spells out. Maybe rephrase to "Stream stopped as capture source was not found". "Capture source was lost" would also be possible, but that would make an assumption again.

Reply:
Changed to "Stream stopped as no capture source was not found".